### PR TITLE
Add a generate sid function and fix bug

### PR DESCRIPTION
**Pull request by EMaksy**
# Description

This pr fixes a small tech debt, by adding a helper function for generating SIDs. 
This will improve the code readability for tests and factories, and also we used faker wrongly, instead of generating three upper-cased letters it just generated one.

### CODE REVIEW

File: assets/js/lib/test-utils/factories/sapSystems.js [hunk 1]
```diff
@@ -26,26 +26,25 @@ export const sapSystemApplicationInstanceFactory = Factory.define(() => ({
...
+  const databaseSid = params.database_sid || generateSid();
   return {
     application_instances: sapSystemApplicationInstanceFactory.buildList(2, {
       sap_system_id: id,
       sid,
     }),
     database_instances: databaseInstanceFactory.buildList(2, {
       database_id: databaseId,
-      sid: faker.string.alphanumeric(3, { casing: 'upper' }),
+      sid: generateSid(),
```
Comment (arbulu89) [inline assets/js/lib/test-utils/factories/sapSystems.js:47]: Shouldn't this and the next one be `databaseSid`, generated on top?
I mean, a SAP system all database instances should have the same SID, right?

**Review by arbulu89 — APPROVED** [review]
Good!
Just one comment.
Did you check if in any other place we generate the SID with UUID or any other faker function? I guess yes, but it wouldn't harm to recheck hehe

File: assets/js/lib/test-utils/factories/sapSystems.js (hunk 1 above)
Comment (EMaksy) [reply assets/js/lib/test-utils/factories/sapSystems.js:47]: Yeah, you are fully right with this!

**Comment by EMaksy** [comment]
Thanks @arbulu89, I took a look and think I caught them all :+1:

**Review by rtorrero — APPROVED** [review]
Nice :-)! looks cleaner indeed :+1:

**Review by CDimonaco — APPROVED** [review]

**Pull request merged**